docs/index.md

```diff
@@ -313,6 +313,25 @@ Finally, the Health Wallet asks the user if they want to save any/all of the sup
 
 For a more seamless user experience when FHIR API connections are already in place, results may also be conveyed through a FHIR API `$health-cards-issue` operation defined [here](../artifacts/operation-patient-i-health-cards-issue.json). For issuers that support SMART on FHIR access, the Health Wallet MAY request authorization with SMART on FHIR scopes (e.g., `launch/patient patient/Immunization.read` for an Immunization use case). This allows the Health Wallet to automatically request issuance of VCs, including requests for periodic updates.
 
+### via “Deep Link”
+
+For a user to import one or more SMART Health Cards to their Health Wallet with one tap or click, issuers can display app specific “deep links”. These are available on most modern operating systems and will open link clicks in a native app if the respective app is installed on the computer or smartphone.
```

Comment:
>https://<<associated domain>><<'/'optional path>>/SMARTHealthCard/#

It's not crystal clear *who* has the option of adding optional paths (I understand it's the server operator but it's implicit here).

Any reason not to instead specify the wallet defines something like a full  "submissions deep link URL, using https or another secure protocol, and ending with `/SMARTHealthCard/#`? Then your syntax can be written relative to the base.

Reply:
Oh yeah, the app will have to define the path, and your latter paragraph is what I meant – the app will define this and we suggest it ends in `/SMARTHealthCard/#`.

Reply:
Yes!